lib/jsdom/browser/default-stylesheet.js

```diff
@@ -128,13 +124,10 @@ q {
     display: inline
 }
 
-q:before {
-    content: open-quote;
-}
-
-q:after {
-    content: close-quote;
-}
+/* nwmatcher does not support :before and :after, so we can't render q
+correctly: https://www.w3.org/TR/html5/rendering.html#phrasing-content-0
```

Comment:
Wrong spec; you want https://html.spec.whatwg.org/multipage/ not the W3C fork. I can fix while merging.